render/hyperscript.js

```diff
@@ -8,8 +8,7 @@ var selectorParser = /(?:(^|#|\.)([^#\.\[\]]+))|(\[(.+?)(?:\s*=\s*("|'|)((?:\\["
 var selectorCache = Object.create(null)
 
 function isEmpty(object) {
-	for (var key in object) if (hasOwn.call(object, key)) return false
-	return true
+	return Object.keys(object).length === 0
```

Comment:
Did you confirm this to be necessary while benchmarking?

What's the tradeoff here? I didn't see anything in the commit message clarifying why it's necessary.

If it's not actually boosting anything, please revert it. It was originally made to use `for ... in` for perf reasons, so I don't want that to change unless it needs to.

Reply:
OK, I'll revert back to the previous code.
The purpose of this change was to reduce the amount of code rather than for performance purposes.
I should have detailed the purpose as it is not directly related to this PR.